src/database/database-factory.ts

```diff
@@ -0,0 +1,9 @@
+import EventDatabase from "./event/event-database";
+
+/**
+ * This defines an Abstract Factory interface for vending database objects.
+ */
+export default interface DatabaseFactory {
+  getEventDatabase(): EventDatabase;
+  // TODO: add getUserDatabase() method
+}
```

Comment:
This might look unnecessary, but as we create more db tables, the Abstract Factory should simplify the process of getting data from them, while only performing configuration once in the Factory implementation